### PR TITLE
UXP 8.0 docs

### PR DESCRIPTION
**Pull request by padmkris123**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/ps_reference/changelog/index.md [hunk 1]
```diff
@@ -28,6 +28,77 @@ contributors:
...
+- Video enhancements: Support added for [Seek Event](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLVideoElement.md)
+- Support for [Navigator.language](https://developer.mozilla.org/en-US/docs/Web/API/Navigator/language) API
+- Support for new HTMLElement APIs - [append](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#appendnodes), [prepend](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#prependnodes) and [replaceChildren](../../uxp-api/reference-js/Global%20Members/HTML%20Elements/HTMLElement.md#replacechildrennodes)
+- [Beta] Support for form reset in [HTMLFormElement](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form)
+
+#### Updated
+- Upgraded Spectrum Web Components to v0.37.0. [More detail](https://github.com/adobe/swc-uxp-wrappers/tree/main?tab=readme-ov-file#spectrum-web-components-uxp-wrappers)
+- [UXP Clipboard.writeText()](../../uxp-api/reference-js/Global%20Members/Data%20Transfers/Clipboard.md#writetexttext) now accepts strings as input to meet standards
+
```
Comment (samgannawayA) [inline src/pages/ps_reference/changelog/index.md:83]: This should move up under the heading:

# Photoshop API Changelog

File: src/pages/ps_reference/changelog/index.md [hunk 2]
```diff
@@ -28,6 +28,77 @@ contributors:
...
+
+#### Fixed
+- XMP now works in UXP Scripts.
+- Plugin crashes while using [fit-content](https://forums.creativeclouddeveloper.com/t/ps-2024-crashes-when-opening-my-uxp-plugin/6840/7) 
+- WebView support for [file selector](https://forums.creativeclouddeveloper.com/t/macos-uxp-webview-for-photoshop-is-missing-file-selector/6843) in MacOS
+- GUID returning hashed empty string
+- Updated missing docs for pseudo-class [defined](../../uxp-api/reference-css/Pseudo-classes/defined.md) (Available since UXP v6.0)
+
+
```
Comment (samgannawayA) [inline src/pages/ps_reference/changelog/index.md:101]: This is a duplicate entry.

**Review by samgannawayA — CHANGES_REQUESTED** [review]

File: src/pages/ps_reference/changelog/index.md (hunk 1 above)
Comment (padmkris123) [reply src/pages/ps_reference/changelog/index.md:63]: Fixed

File: src/pages/ps_reference/changelog/index.md (hunk 2 above)
Comment (padmkris123) [reply src/pages/ps_reference/changelog/index.md]: Fixed.

**Pull request merged**